python/langsmith/evaluation/_runner.py

```diff
@@ -1438,7 +1460,7 @@ def _score(
         (e.g. from a previous prediction step)
         """
         with ls_utils.ContextThreadPoolExecutor(
-            max_workers=max_concurrency
+            max_workers=max_concurrency or 1
```

Comment:
This will change default behavior, no? Do we want to do in a separate PR?

Reply:
cc @hinthornw, dont think i made this one. was this on purpose?